monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+
+**Case 3:** En el 2014 Christos y Markella Sourovelis, [padres de un joven de 22 años, tuvieron una gran sorpresa](https://edition.cnn.com/2014/09/03/us/philadelphia-drug-bust-house-seizure/) después de que la policía llegara a la puerta de su casa arrestando a su hijo por la venta de $40 USD en heroína. Las autoridades acusaron al hijo de vender drogas afuera de su casa, lo cual sus padres negaron de manera enfática tener algún conocimiento de ello. Dos meses después, la policía regreso e incauto su casa, la energía fue suspendida, las puertas selladas y la familia estuvo forzada a estar en la calle durante un día. 
+ 
+La policía y la fiscalía alegaron de manera rotunda que su hijo había sido visto vendiendo drogas afuera de la casa siendo esto sujeto a leyes estatales de decomiso civil. Después de una batalla excesiva en la corte y ocho días viviendo en casa de familiares, a los Sourovelis les permitieron el ingreso a su casa con la condición de prohibirle el ingreso permanentemente a su hijo. 
+
+Ten en cuenta que esto es sola una pequeña fracción de [los billones incautados cada año](https://harvardlawreview.org/2018/06/how-crime-pays-the-unconstitutionality-of-modern-civil-asset-forfeiture-as-a-tool-of-criminal-law-enforcement/) de manera ilegal.
+
+## **Como Monero puede servir y proteger a las personas de las injustas leyes de decomiso civil**  
+
+Monero, es una criptomoneda segura, rápida y privada, sirve para al beneficio de aquellos que podrían caer presos de las violaciones constitucionales y a la interpretación de la ley. 
+
+Con respecto a las injustas leyes de decomiso civil de los EEUU (y a una [minoría de jurisdicciones](https://en.wikipedia.org/wiki/Asset_forfeiture#European_Union)), Monero sirve para prevenir lo siguiente: 
+
+1. Comportamiento codicioso del gobierno y de la policía.
+2. Castigos arbitrarios y/o desproporcionados. 
+
+Veamos como Monero puede ayudar a prevenir ambos. 
+
+### **Comportamiento policial y gubernamental codicioso** 
+
+Como señalamos sobre el programa norteamericano de "intercambio equitativo", empleado por la mayoría de estados, el sistema incentiva el comportamiento abusivo y codicioso policíaco, permitiendo la operación libre de impunidad de parte de las agencias gubernamentales y funcionarios del gobierno. Activos líquidos como el dinero, joyas, vehículos y casas, ofrecen un gancho tentador para los funcionarios, aumentando la presunción de actividad ilegal de parte de la policía. El dinero en efectivo y las joyas son fácilmente convertidos en instrumentos y herramientas policiales. 

```

Comment:
+ ...empleado por _**algunos**_ estados,...

The original text doesn't say _all_, but _some_ (algunos).

Reply:
done